nextflow.config

```diff
@@ -95,15 +95,15 @@ process {
 
 report {
```

Comment:
I'd suggest removing these from the config. The decision about where to store the report, timeline and trace and what to name them can be made when the pipeline is run using the `-with-report`, `-with-trace` and `-with-timeline` flags.